example_docs/docs/index.rst

```diff
@@ -16,15 +16,14 @@ Qiskit sphinx theme |version| documentation
    sphinx_guide/structural
 
 .. toctree::
-  :hidden:
   :caption: API
+  :maxdepth: 2
 
    Autodoc <sphinx_guide/autodoc>
-   Super duper duper duper duper duper long title to test line wrapping <sphinx_guide/long_title>
```

Comment:
This is already down below in the Special Elements section. It was confusing to duplicate in two places.